src/Console/Commands/Model/Apply/UIBehavioursApplyService.cs

```diff
@@ -23,8 +24,18 @@ public UIBehavioursApplyService(IApiClient apiClient)
                     UIEntity data)
         {
             var definition = await GetDefinitionForEntity(tenant, environment, entity).ConfigureAwait(false);
-            var patch = new JsonPatchDocument()
-                .Replace("/behaviours", data.EntityBehaviours.ToArray());
+
+            var patch = new JsonPatchDocument();
+
+            var attributeBehaviours = data.EntityBehaviours.Where(a => !string.IsNullOrEmpty(a.Element)).ToList();
```

Comment:
entityBehaviours instead of attributeBehaviours

Reply:
Reviewed